hawkular-agent-core/src/main/java/org/hawkular/agent/monitor/util/Util.java

```diff
@@ -389,4 +392,17 @@ public static String extractDockerContainerIdFromString(String s) {
 
         return null;
     }
+
+    public static Manifest manifestFromClass(Class clazz) throws IOException {
+        String filename = clazz.getName().replace('.', '/') + ".class";
+        String classPath = clazz.getClassLoader().getResource(filename).toString();
+        if (!classPath.startsWith("jar")) {
+            return null;
+        }
+        String manifestPath = classPath.substring(0, classPath.lastIndexOf('!') + 1) + File.separator + JarFile.MANIFEST_NAME;
+        InputStream stream = new URL(manifestPath).openStream();
+        Manifest manifest = new Manifest(stream);
+        stream.close();
+        return manifest;
```

Comment:
This seems overkill. Can we just remove this Util method and in AgentCoreEngine simply do:

```
log.infoTypeAndVersion(this.getClass().getPackage().getImplementationTitle(),
    this.getClass().getPackage().getImplementationVersion());
```